static/app/views/performance/landing/index.tsx

```diff
@@ -69,7 +72,33 @@ export function PerformanceLanding(props: Props) {
 
   const {teams, initiallyLoaded} = useTeams({provideUserTeams: true});
 
-  const currentLandingDisplay = getCurrentLandingDisplay(location, projects, eventView);
+  const hasMounted = useRef(false);
```

Comment:
I wonder if we can wrap this into some `useIsMounted` hook?

Reply:
We can if we use this pattern more 👍